### PR TITLE
cmd/snappy,daemon,snap,snappy: remove SetActive from parts

### PR DESCRIPTION
**Pull request by chipaca**
This branch removes SetActive from the parts interface.

from #399

### CODE REVIEW

**Comment by chipaca** [comment]
:+1: fwiw :-)

File: snappy/kernel_os.go [hunk 1]
```diff
@@ -222,13 +222,14 @@ func SyncBoot() error {
 		return fmt.Errorf("failed to run SyncBoot: %s", err)
 	}
 
+	overlord := &Overlord{}
```
Comment (zyga) [inline snappy/kernel_os.go]: Why Overload directly vs getActivator?

**Comment by zyga** [comment]
Just one question, otherwise +1.

BTW: what is the false argument on .activate() and .deactivate()?

File: snappy/kernel_os.go (hunk 1 above)
Comment (chipaca) [inline snappy/kernel_os.go]: hah! I was wondering the same, but the other way around: why not just use an overlord there. But I'm fine with landing this as is and waiting for @mvo5's answer/patch later :-)

Comment (zyga) [inline snappy/kernel_os.go]: AFAIR we don't want to sprinkle &Overlord{} everywhere. There will be only one instance.

**Comment by chipaca** [comment]
something about running the hooks or not (you don't run the hooks when activating from u-d-f)

**Pull request merged**